application/clicommands/SendCommand.php

```diff
@@ -114,11 +118,12 @@ public function problemAction()
             $ackMessage = "JIRA issue $key has been created";
         } else {
             $key = $issue->key;
-            $currentStatus = isset($issue->fields->icingaStatus) ? $issue->fields->icingaStatus : null;
+            $icingaStatus = $config->get('ui', 'field_icingaStatus', 'customfield_19220');
```

Comment:
Guess something went wrong here. It's not UI, and we cannot hard-code custom field identifiers - they are different on different Jira instances.